app/views/transactions/_transaction.html.erb

```diff
@@ -1,21 +1,32 @@
 <tr>
+  <%
+    other_account, amount = nil, nil
```

Comment:
I don't like the name other_account but I can't come up with something better :confused: